app/assets/javascripts/discourse/templates/components/login-buttons.hbs

```diff
@@ -1,3 +1,10 @@
 {{#each buttons as |b|}}
   <button class="btn btn-social {{b.name}}" {{action "externalLogin" b}}>{{b.title}}</button>
 {{/each}}
+
+{{#if siteSettings.enable_local_logins_via_email}}
```

Comment:
The entire component is hidden if the length of `buttons` is `0`

Reply:
weird I remember coding something for it... 🤔